init.lua

```diff
@@ -1,570 +1,628 @@
 if not minetest.settings:get_bool("enable_damage") then
-	minetest.log("warning", "[stamina] Stamina will not load if damage is disabled (enable_damage=false)")
-	return
+    minetest.log("warning", "[stamina] Stamina will not load if damage is disabled (enable_damage=false)")
```

Comment:
Please use tabs to indent. It touches almost all lines, which makes bisecting very annoying. Luanti and a majority of mods use tabs, very likely due to https://docs.luanti.org/for-engine-devs/lua-code-style-guidelines/ .